File: math_functions.py
#Functions

#To get min from array
x = min(4, 8, 60)

#To get max from array
y = max(4, 8, 60)

#To get absolute number
x = abs(-6)

#2 Power of 8
x = pow(8, 2)

#Some math functions

#To be able to use math functions, we have to import math
import math

#Square of number
x = math.sqrt(4)
print(x)

#Round up
x = math.ceil(2.2)

#Round down
y = math.floor(2.2)

#Pi value (3.14...)
x = math.pi

print(x)

#Using numpy functions
import numpy

a = [1,2,3,4,5,6,7,8]

#Get mean from array
x = numpy.mean(a)

print(x)

#Get media form array
x = numpy.median(a)

print(x)

#Standard desviation from array
x = numpy.std(a)

print(x)

#Percentil
x = numpy.percentile(a, 90)

print(x)

#Using stats functions
from scipy import stats

x = stats.mode(a)

print(x)

File: math_functions.R
# Functions

# Vector
numbers = c(4, 8, 60, 60, 80)

# To get min from array
x = min(numbers)

# To get max from array
y = max(numbers)

# To get absolute number
x = abs(-6)

#2 Power of 8
x = 2^8

# Square of number
x = sqrt(4)
print(x)

# Round up
ceiling(2.2)

# Round down
y = floor(2.2)

# Pi value (3.14...)
print(pi)

# Get mean from vector
x = mean(numbers)

print(x)

# Get media form array
x = median(numbers)

print(x)

# Standard desviation from array
x = sd(numbers)

print(x)

# Percentil
quantile(numbers, probs = c(.25, .5, .75))

print(x)

# Median
x = median(numbers)

# Mean
x = mean(numbers)

print(x)

# Function to get Mode
mde <- function(c) {
  u <- unique(c)
  u[which.max(tabulate(match(c, u)))]
}

print(mde(numbers))